fhir-server/src/main/java/au/csiro/pathling/fhirpath/function/EmptyFunction.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2018-2020, Commonwealth Scientific and Industrial Research
+ * Organisation (CSIRO) ABN 41 687 119 230. Licensed under the CSIRO Open Source
+ * Software Licence Agreement.
+ */
+
+package au.csiro.pathling.fhirpath.function;
+
+import static au.csiro.pathling.fhirpath.function.NamedFunction.checkNoArguments;
+import static au.csiro.pathling.fhirpath.function.NamedFunction.expressionFromInput;
+import static org.apache.spark.sql.functions.max;
+import static org.apache.spark.sql.functions.when;
+
+import au.csiro.pathling.fhirpath.FhirPath;
+import au.csiro.pathling.fhirpath.NonLiteralPath;
+import java.util.function.Function;
+import javax.annotation.Nonnull;
+import org.apache.spark.sql.Column;
+import org.hl7.fhir.r4.model.Enumerations.FHIRDefinedType;
+
+/**
+ * This function returns true if the input collection is empty.
+ *
+ * @author John Grimes
+ * @see <a href="https://pathling.csiro.au/docs/fhirpath/functions.html#empty">first</a>
```

Comment:
empty NOT first (in the link name)

Reply:
See 124fd90795650e34b41eb302bf3f0e94d743da4f.